src/wp-content/themes/twentynineteen/sass/elements/_elements.scss

```diff
@@ -40,6 +40,8 @@ a:focus {
 	outline: thin;
 	outline-style: dotted;
 	text-decoration: underline;
+	text-decoration-skip-ink: auto;
+ 	text-decoration-thickness: 2px;
```

Comment:
Small formatting change: removing the space before the tab on the `text-decoration-thickness` line

Reply:
Removed it :wave: